nilearn/decoding/searchlight.py

```diff
@@ -50,6 +50,9 @@ def search_light(X, y, estimator, A, scoring=None, cv=None, n_jobs=-1,
         adjacency matrix. Defines for each feature the neigbhoring features
         following a given structure of the data.
 
+    groups : array-like
```

Comment:
You can take optional beside array-like to short the line.

array-like, optional